System/Random.hs

```diff
@@ -445,11 +438,18 @@ Minimal complete definition: 'randomR' and 'random'.
 
 -}
 
-class Random a where
-  randomRM :: MonadRandom g m => (a, a) -> g -> m a
 
-  randomM :: MonadRandom g m => g -> m a
+class Uniform a where
+  uniform :: MonadRandom g m => g -> m a
 
+class UniformRange a where
+  uniformR :: MonadRandom g m => (a, a) -> g -> m a
```

Comment:
Here we have question of semantics. What should  `uniformR (a,b)` do for `a > b`? `random`'s position is that it's unspecified. `mwc-random` made choice `uniformR (a,b) = uniformR (b,a)`

Reply:
I think that is a fine choice. It is definitely better than throwing an error. We could of course return a `Maybe`, but for consistency sake with other libraries and current `randomR` I think: `uniformR (a,b) = uniformR (b,a)` is the right way to go.